cspell/.cspell.json

```
@@ -70,6 +70,7 @@
         "planing"
     ],
     "words": [
+        "Yushi"
```

```suggestion
        "Yushi",
```
The trailing comma is needed.